sdk/storage/Azure.Storage.Files.Shares/src/Models/FileRequestConditions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.Files.Shares.Models
+{
+    /// <summary>
+    /// Specifies lease access conditions for a file.
+    /// </summary>
+    public class FileRequestConditions
```

Comment:
I chose to create a FileRequestConditions object now in case the Files service ever gets other request conditions (IfMatch, IfNoneMatch, etc.)